RG.CodeAnalyzer/RG.CodeAnalyzer.Test/AddMissingNamedArgumentsRefactoringTests.cs

```diff
@@ -0,0 +1,333 @@
+			
+			int position = test.IndexOf("DoSomething(1");
+			TextSpan span = new TextSpan(position, "DoSomething".Length);
+
+			List<CodeAction> actions = GetRefactorings(document, span);
+
+			Assert.IsTrue(actions.Any(a => a.Title.Contains("Add missing named arguments")), 
+				"Should offer 'Add missing named arguments' refactoring for method calls");
+		}
+
+		[TestMethod]
+		public void TestMethodInvocation_ApplyRefactoring_AddsNamedArguments() {
+			string test = @"
+using System;
+
+namespace ConsoleApplication1
+{
+	class TypeName
+	{
+		public void Foo() {
+			DoSomething(1, ""test"");
+		}
+
+		private void DoSomething(int id, string name) { }
+	}
+}";
+			string expected = @"
+using System;
+
+namespace ConsoleApplication1
+{
+	class TypeName
+	{
+		public void Foo() {
+			DoSomething(
+				id: _,
+				name: _
+			);
+		}
+
+		private void DoSomething(int id, string name) { }
+	}
+}";
+			Document document = CreateDocument(test, LanguageNames.CSharp);
+			
+			int position = test.IndexOf("DoSomething(1");
+			TextSpan span = new TextSpan(position, "DoSomething".Length);
+
+			List<CodeAction> actions = GetRefactorings(document, span);
+			CodeAction? action = actions.FirstOrDefault(a => a.Title.Contains("Add missing named arguments"));
+
+			Assert.IsNotNull(action, "Should find the refactoring action");
+
+			Document newDocument = ApplyRefactoring(document, action);
+			string newCode = GetStringFromDocument(newDocument);
+
+			Assert.AreEqual(NormalizeWhitespace(expected), NormalizeWhitespace(newCode));
+		}
+
+		[TestMethod]
+		public void TestObjectCreation_MultipleOverloads_OffersMultipleRefactorings() {
+			string test = @"
+using System;
+
+namespace ConsoleApplication1
+{
+	class Person {
+		public Person(int id) { }
+		public Person(int id, string firstName) { }
+		public Person(int id, string firstName, string lastName) { }
+	}
+
+	class TypeName
+	{
+		public void Foo() {
+			var person = new Person(1);
+		}
+	}
+}";
+			Document document = CreateDocument(test, LanguageNames.CSharp);
+			
+			int position = test.IndexOf("new Person");
+			TextSpan span = new TextSpan(position, "new Person".Length);
+
+			List<CodeAction> actions = GetRefactorings(document, span);
+
+			var namedArgActions = actions.Where(a => a.Title.Contains("Add missing named argument")).ToList();
+			Assert.AreEqual(3, namedArgActions.Count, 
+				$"Should offer 3 refactorings for 3 overloads. Found {namedArgActions.Count}");
+			
+			Assert.IsTrue(namedArgActions.Any(a => a.Title == "Add missing named argument (Int32)"),
+				"Should offer refactoring for Person(int id)");
+			Assert.IsTrue(namedArgActions.Any(a => a.Title == "Add missing named arguments (Int32, String)"),
+				"Should offer refactoring for Person(int id, string firstName)");
+			Assert.IsTrue(namedArgActions.Any(a => a.Title == "Add missing named arguments (Int32, String, String)"),
+				"Should offer refactoring for Person(int id, string firstName, string lastName)");
```

Comment:
Change the implementation to show parameter names instead of parameter types in the titles